warehouse/validations/helper.go

```diff
@@ -59,7 +58,7 @@ func parseOptions(req json.RawMessage, v interface{}) error {
 }
 
 func randomString() string {
-	return strings.ReplaceAll(uuid.Must(uuid.NewV4()).String(), "-", "")
+	return strings.ReplaceAll(misc.FastUUID().String(), "-", "")
```

Comment:
There is probably a faster way to do this by converting the UUID bytes directly to runes. If this is used in a performance-critical path, it is probably worth it ( a benchmark will be advised)

Reply:
We can use our own, modified version of encodeHex(dst []byte, uuid UUID) https://github.com/google/uuid/blob/44b5fee7c49cf3bcdf723f106b36d56ef13ccc88/uuid.go#L200

A single call to [hex.Encode](https://pkg.go.dev/encoding/hex#Encode) should be enough.

Reply:
```suggestion
	var buf [32]byte
	 hex.Encode(buf[:], misc.FastUUID())
	return string(buf[:])
```

Reply:
Make sense. Since we are ignoring `-`. It's better to use this.

Reply:
Done.